frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -1107,6 +1112,45 @@ function ContextMenuInner(propsWithInputRef: Props) {
     maybeViewport,
   } = props;
 
+  const segmentIdAtPosition = globalPosition != null ? getSegmentIdForPosition(globalPosition) : 0;
+  const { visibleSegmentationLayer, volumeTracing } = props;
+  const hasNoFallbackLayer =
+    visibleSegmentationLayer != null &&
+    "fallbackLayer" in visibleSegmentationLayer &&
+    visibleSegmentationLayer.fallbackLayer == null;
+  const [segmentVolume, boundingBoxInfo] = useFetch(
+    async () => {
+      if (contextMenuPosition == null) return [];
+      if (visibleSegmentationLayer != null && volumeTracing != null && hasNoFallbackLayer) {
```

Comment:
in the else-branch, nothing is returned, right?

Reply:
i think it should be resolved with the current changes!